tests/test_tpl.py

```diff
@@ -1,6 +1,18 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
+from copier import run_copy
 
 def test_tpl():
     assert True
+
+def helper(dest):
+   run_copy("https://github.com/jupyterlab/extension-template.git", dest, 
+            data={"kind": "frontend", "author_name": "tester"}, unsafe=True)
```

Comment:
Ok I figure out the issue. The correct syntax is:


```suggestion
   run_copy(
       "https://github.com/jupyterlab/extension-template",
       dest,
       defaults=True,
       data={"author_name": "tester", "repository": "dummy"},
       unsafe=True
   )
```